framework/plugins/plugin.go

```diff
@@ -159,6 +160,51 @@ func resolvePlugins(pluginsDir, assetsDir, downloadsDir string, osArch osarch.OS
 	return nil, errors.New(strings.Join(errStringsParts, "\n"+strings.Repeat(" ", pluginsinternal.IndentSpaces)))
 }
 
+func groupByProductAndGroup(plugins []godellauncher.SinglePluginParam) map[string][]godellauncher.SinglePluginParam {
```

Comment:
Would rename this to `groupByGroupAndProduct` just to keep this consistent (the other variables in this function go by `GroupAndProduct`)